packages/terra-consumer-nav/src/components/user-profile/UserProfile.scss

```diff
@@ -10,7 +10,7 @@
       background: none;
       background-attachment: none;
       background-size: none;
-      border-top: 3px solid var(--terra-consumer-body-background-color, #c7d4ea);
+      border-top: 3px solid var(--terra-consumer-profile-link-border-top, #c7d4ea);
```

Comment:
this is limited to the border-color.... Either change the name to border color or allow it to modify the whole border. e.g.
```csss
border-top: var(--terra-consumer-profile-link-border-top, 3px solid #c7d3ea);
```